README.md

```diff
@@ -1,2 +1,60 @@
 # QueryBuilder
 A Query builder to query the Knowledge Store and generate datafiles that can be visualized with the Storyteller interface.
+
+
+
+The proposed system will have 3 main components:
+
+0. KnowledgeStore
+    - RDF triple store. The NewsReader KnowledgeStore is a scalable, fault-tolerant, and Semantic Web grounded storage system to jointly store, manage, retrieve, and semantically query, both structured and unstructured data (see https://knowledgestore.fbk.eu/).
+1. QueryBuilder
+    - written in React.js 
+    - gets a list of all possible components, agents and events from the server component (or KnowledgeStore, tbd later)
```

Comment:
- Shouldn't 'components, agents and events' be the same as in line 13: 'concepts, actors and events'
- also, it's probably wise to reserve the word 'components' strictly for React.component (I know I started this particular confusion myself)